pkg/kms/kms.go

```diff
@@ -3,22 +3,19 @@ package kms
 import (
 	"fmt"
 
-	"github.com/enj/kms/api/v1beta1"
-	"github.com/enj/kms/pkg/encryption"
+	"github.com/enj/citadel/api/v1beta1"
+	"github.com/enj/citadel/pkg/api"
+	"github.com/enj/citadel/pkg/encryption"
 
 	"golang.org/x/net/context"
 )
 
-const (
-	version        = "v1beta1"
-	runtimeName    = "kms_cmd"
-	runtimeVersion = "0.0.1" // TODO embed git sha on build
-)
+const kmsAPIVersion = "v1beta1"
 
 var apiVersionResponse = &v1beta1.VersionResponse{
```

Comment:
I assume this is what you are renaming later?

Reply:
as its still `kms` in this file.

Reply:
This `kms` is correct because it refers to the Kube API only.

Reply:
+1 for the rename, then.

😄